packages/web-react/src/components/UNSTABLE_ProductLogo/stories/UNSTABLE_ProductLogo.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import ReadMe from '../README.md';
+import UNSTABLE_ProductLogo from '../UNSTABLE_ProductLogo';
+
+const meta: Meta<typeof UNSTABLE_ProductLogo> = {
+  title: 'Components/UNSTABLE_ProductLogo',
```

Comment:
Sorry... I have to rename to experimental a also updated whole stories to correctly interact with isInverted option